### PR TITLE
v4.1.0 Adding Builder Kit

### PR DESCRIPTION
**Pull request by aweibel**
Releasing v4.1.0, adding SLDS Pattern - Builder Sketch document

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,30 +26,30 @@ Download the most recent version of [Sketch](https://www.sketchapp.com/).
...
 
-**Welcome to the Summer ’18 Sketch Library document**
+**Welcome to the Summer ’18 Sketch Library kit**
 
-The Salesforce UX team has made quite a few changes to this document since the last release. We made optimizations so you can more efficiently create Salesforce Lightning interfaces. In the instructions below, we’ll cover the basics and some tips for you to get started with this Sketch Library document. You can still use this document just like you have in previous releases, but the Salesforce UX team believes this file is best used as a Sketch Library. What does that mean and why should you use it that way? We’ll answer those questions below.
+The Salesforce UX team has made quite a few changes to this kit since the last release. We made optimizations so you can more efficiently create Salesforce Lightning interfaces. In the instructions below, we’ll cover the basics and some tips for you to get started with this Sketch Library kit. You can still use this kit just like you have in previous releases, but the Salesforce UX team believes this file is best used as a Sketch Library. What does that mean and why should you use it that way? We’ll answer those questions below.
```
Comment (engai) [inline README.md]: Looks like theres more than 1 now, so - "believes these files are best used as Sketch Libraries"

Comment (engai) [reply README.md]: why should you use **them** that way

File: README.md [hunk 2]
```diff
@@ -26,30 +26,30 @@ Download the most recent version of [Sketch](https://www.sketchapp.com/).
...
+**Why use these files as a Sketch Libraries?**
 
-Sketch has released a feature that allowed any Sketch file’s symbols to be accessed and used across all of your Sketch documents. They called this feature Sketch Libraries. You can now have all SLDS components available from Sketch’s symbols menu in any file you open. Another powerful feature is that Sketch Libraries will automatically update your designs when the Library is updated. Your designs will never be out of date if they’re linked to this Sketch Library document.
+Sketch has released a feature that allowed any Sketch file’s symbols to be accessed and used across all of your Sketch documents. They called this feature Sketch Libraries. You can now have all SLDS components and patterns available from Sketch’s symbols menu in any file you open. Another powerful feature is that Sketch Libraries will automatically update your designs when the Library is updated. Your designs will never be out of date if they’re linked to this Sketch Library document.
 <br />
 <br />
 
-**How to use this document as a Sketch Library in two quick steps**
+**How to use these Sketch documents as a Sketch Libraries in two quick steps**
```
Comment (engai) [inline README.md]: I would emphasize here that the builders library is dependent on the components library. It's implied in the architecture section below, but would be great to explicitly call that out here for first time users.

Comment (aweibel) [reply README.md]: It's not dependent per say but they work together if you have both. This is outlined in the "Lightning Design System Sketch Library Architecture" section.

File: README.md (hunk 1 above)
Comment (aweibel) [reply README.md]: Good catch. Fixed.

File: README.md (hunk 2 above)
Comment (engai) [reply README.md]: Gotcha. My bad, I had thought that the Builders library was dependent on some of the symbols from the components library.

**Comment by engai** [comment]
LGTM 👍

**Pull request merged**